internal/configs/version2/template_helper.go

```diff
@@ -114,7 +125,7 @@ func buildListenDirective(port string, proxyProtocol bool, listenType listenerTy
 	return directive
 }
 
-func buildTransportListenDirective(listenType listenerType, port string, ssl *StreamSSL, udp bool) string {
+func buildTransportListenDirective(listenType ipType, port string, ssl *StreamSSL, udp bool) string {
```

Comment:
Can you also rename the `listenType` variable, to ipType, like in the `buildListenDirective` function